include/verilated_dynamic_scheduler.h

```diff
@@ -0,0 +1,362 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Code available from: https://verilator.org
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+#ifndef VERILATOR_VERILATED_DYNAMIC_SCHEDULER_H_
+#define VERILATOR_VERILATED_DYNAMIC_SCHEDULER_H_
+
+#ifndef VERILATOR_VERILATED_H_INTERNAL_
+#error "verilated_dynamic_scheduler.h should only be included by verilated.h"
+#endif
+
+// Dynamic scheduler-related includes
+#include <vector>
+#include <map>
+#include <functional>
+#include <utility>
+#include <queue>
+#include <unordered_map>
+#include <unordered_set>
+#include <functional>
```

Comment:
Why aren't we using `clang-format` for that?

Reply:
Because clang-format at one point sorted some things that it shouldn't.  It's possible these could be cleaned up e.g. by adding blank lines at the right spots, if you want to make a separate pull request to do that it would be welcome.

Reply:
Sorted.